broker/broker-route-manager/src/main/java/org/codice/ddf/broker/route/manager/DynamicRouteDeployer.java

```diff
@@ -32,15 +32,34 @@
 public class DynamicRouteDeployer implements ArtifactInstaller {
   private static final Logger LOGGER = LoggerFactory.getLogger(DynamicRouteDeployer.class);
 
-  private final CamelContext camelContext;
+  private static Map<File, List<RouteDefinition>> processedFiles = new ConcurrentHashMap<>();
 
-  private Map<File, List<RouteDefinition>> processedFiles = new HashMap<>();
+  private final CamelContext camelContext;
 
   public DynamicRouteDeployer(CamelContext camelContext) {
     this.camelContext = camelContext;
   }
 
-  public void init() {}
+  public void init() throws Exception {
+    if (processedFiles.isEmpty()) {
+      return;
+    }
+    processedFiles
+        .entrySet()
+        .stream()
+        .forEach(
+            entry ->
+                entry
+                    .getValue()
+                    .forEach(
+                        routeDefinition -> {
+                          try {
+                            camelContext.addRouteDefinition(routeDefinition);
```

Comment:
❓ have you considered simply calling `camelContext.addRouteDefinitions() with the list associated with each entry in the map?

Reply:
Yes, I think that's a better solution.